packages/sdk-rtl/src/transport.ts

```diff
@@ -470,32 +471,35 @@ function bufferString(val: any) {
  */
 export function sdkError(response: any) {
   if (typeof response === 'string') {
-    return new Error(response)
+    return new LookerSdkError(response)
   }
   if ('error' in response) {
     const error = response.error
     if (typeof error === 'string') {
-      return new Error(error)
+      return new LookerSdkError(error)
     }
     // Try to get most specific error first
     if ('error' in error) {
       const result = bufferString(error.error)
-      return new Error(result)
+      return new LookerSdkError(result)
     }
     if ('message' in error) {
-      return new Error(response.error.message.toString())
+      return new LookerSdkError(response.error.message.toString(), {
+        errors: error.errors,
```

Comment:
what happens if `error.errors` is not defined?

Reply:
`errors` will be an empty list if `error.errors` is not defined.